VPS docker deployment does not work

already git clone the hospitalrun-server, change the DOMAIN-NAME in the docker-compose.yml  but when i run docker-compose up --build -d  crash with the following error.

ERROR: Service 'hospitalrun' failed to build: The command '/bin/sh -c npm install --loglevel silent' returned a non-zero code: 1

I looked into it and there's this error from `npm install`.

```
npm WARN deprecated node-uuid@1.4.8: Use uuid module instead
npm ERR! git clone --template=/root/.npm/_git-remotes/_templates --mirror git@github.com:blakeembrey/router.git /root/.npm/_git-remotes/git-github-com-blakeembrey-router-git-router-engine-45f457fb: undefined
npm ERR! git clone --template=/root/.npm/_git-remotes/_templates --mirror git@github.com:blakeembrey/router.git /root/.npm/_git-remotes/git-github-com-blakeembrey-router-git-router-engine-45f457fb: undefined
npm ERR! Linux 4.9.60-linuxkit-aufs
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "install"
npm ERR! node v6.13.0
npm ERR! npm  v3.10.10
npm ERR! code ENOGIT

npm ERR! not found: git
npm ERR! 
npm ERR! Failed using git.
npm ERR! This is most likely not a problem with npm itself.
npm ERR! Please check if you have git installed and in your PATH.

npm ERR! Please include the following file with any support request:
npm ERR!     /usr/src/app/npm-debug.log
ERROR: Service 'hospitalrun' failed to build: The command '/bin/sh -c npm install' returned a non-zero code: 1
```

Which means that `npm install` does not have access to the `git` command. That's because it's not included in the slim version of the node docker image. To fix this you can edit the first line in `Dockerfile` and change it to `FROM node:boron` (remove -slim).

I'm not sure what is the proper fix (I saw this project for the first time today) but I hope this helps.

--- a/Dockerfile
+++ b/Dockerfile
@@ -4,12 +4,13 @@ LABEL maintainer Mofesola Babalola <me@mofesola.com>
 #Get required applications
 ENV DEBIAN_FRONTEND noninteractive
 
+RUN apt-get update && apt-get install -y git
+
 #Create App Directory
 WORKDIR /usr/src/app
 
 #Install Dependencies
 COPY package.json /usr/src/app
-RUN apt-get update && apt-get install -y git
 RUN npm install --loglevel silent
 
 COPY . /usr/src/app

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,4 +1,4 @@
-FROM node:boron-slim
+FROM node:boron
 LABEL maintainer Mofesola Babalola <me@mofesola.com>
 
 #Get required applications

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,4 +1,4 @@
-FROM node:boron
+FROM node:boron-slim
 LABEL maintainer Mofesola Babalola <me@mofesola.com>
 
 #Get required applications
@@ -9,6 +9,7 @@ WORKDIR /usr/src/app
 
 #Install Dependencies
 COPY package.json /usr/src/app
+RUN apt-get update && apt-get install -y git
 RUN npm install --loglevel silent
 
 COPY . /usr/src/app

